front-end/src/App.js

```diff
@@ -2,56 +2,57 @@ import React, { useState, useEffect } from "react";
 import "./App.css";
 import SearchBar from "./components/toolbar/SearchBar";
 import { api } from "./util/api";
-import SortableRow from "./components/list/SortableRow";
-import FIXTURE from "./util/fixture.json";
+import CraftspersonRow from "./components/list/CraftspersonRow";
 import { sortByNumberOfMentees, sortByCraftspeopleWithoutMentor } from "./util/sorting";
 import { filter } from "./util/filtering";
-import Header from "./components/header/Header";
 import ButtonToolbar from 'react-bootstrap/ButtonToolbar'
 import ToggleButton from 'react-bootstrap/ToggleButton';
 import ToggleButtonGroup from 'react-bootstrap/ToggleButtonGroup';
 
 function App() {
+    const defaultSort = sortByNumberOfMentees
```

Comment:
This is very good, I was thinking yesterday about having a `function` in the `sorting.js` called `defaultSort()` that called the `sortByNumberOfMentees`. I think this is clear, but I'm having a little trouble of thinking who's responsibility is to set the default if the `App` or the `sorting.js`